### PR TITLE
rename the file from blend360_abm_density_score to blend360_abm_densi…

### PR DESCRIPTION
**Pull request by hafizasyraf72**
…ty_score_new

### CODE REVIEW

File: Clients/Blend360/Analytics Script/ABM Score/blend360_abm_density_score_new.sql [hunk 1]
```diff
@@ -0,0 +1,1801 @@
...
+  filled_highest_month_awareness_score AS highest_month_score_awareness,
+  filled_highest_month_engagement_score AS highest_month_score_engagement
+FROM filled_data
+)
+SELECT 
+  _with_lag.*,
+  CASE 
+    WHEN _awareness_score IS NOT NULL AND prev_awareness_score IS NOT NULL THEN _awareness_score - prev_awareness_score 
+    ELSE NULL 
+  END AS awareness_score_change,
```
Comment (izztsfn) [inline Clients/Blend360/Analytics Script/ABM Score/blend360_abm_density_score_new.sql:1422]: I think you can replace this CASE statement with IF to make it much simpler and readable. Example:
image

Then you can update the rest of the following CASE statement as highlighted in the pic above

**Review by izztsfn — CHANGES_REQUESTED** [review]
Overall formatting looks good! Just some minor changes to make

File: Clients/Blend360/Analytics Script/ABM Score/blend360_abm_density_score_new.sql [hunk 2]
```diff
@@ -0,0 +1,1801 @@
...
+  acc.icp_tier,
+  acc.customer_segment,
+  'Form Fill - Contact Us' AS _engagement,
+  _email,
+  NULL AS _click_impression,
+  _contactusconversiontitle AS _campaign,
+  CAST(FORMAT_DATE('%Y-%m-%d', PARSE_DATE('%m/%d/%Y', _contactusconversiondate)) AS DATE) AS _timestamp,
+  '' AS _Page_Category,
+  '' AS _page_group,
+  ROW_NUMBER() OVER(
```
[the remark this answers is not in the document]
Comment (hafizasyraf72) [reply Clients/Blend360/Analytics Script/ABM Score/blend360_abm_density_score_new.sql:315]: done

File: Clients/Blend360/Analytics Script/ABM Score/blend360_abm_density_score_new.sql [hunk 3]
```diff
@@ -0,0 +1,1801 @@
...
```
[the remark this answers is not in the document]
Comment (hafizasyraf72) [reply Clients/Blend360/Analytics Script/ABM Score/blend360_abm_density_score_new.sql:341]: done

[the remark this answers is not in the document]
Comment (izztsfn) [reply Clients/Blend360/Analytics Script/ABM Score/blend360_abm_density_score_new.sql:341]: I do not see any new commits. Have you pushed it? @hafizasyraf72

File: Clients/Blend360/Analytics Script/ABM Score/blend360_abm_density_score_new.sql (hunk 1 above)
Comment (hafizasyraf72) [reply Clients/Blend360/Analytics Script/ABM Score/blend360_abm_density_score_new.sql]: done as well. you can verify already ya @izztsfn

**Review by izztsfn — APPROVED** [review]
Looks good!

**Pull request merged**